src/components/MoneyRequestReportView/MoneyRequestReportTransactionsNavigation.tsx

```diff
@@ -12,25 +13,47 @@ import getEmptyArray from '@src/types/utils/getEmptyArray';
 
 type MoneyRequestReportRHPNavigationButtonsProps = {
     currentReportID: string;
+    parentReportID?: string;
+    policyID?: string;
 };
 
-function MoneyRequestReportTransactionsNavigation({currentReportID}: MoneyRequestReportRHPNavigationButtonsProps) {
+function MoneyRequestReportTransactionsNavigation({currentReportID, parentReportID, policyID}: MoneyRequestReportRHPNavigationButtonsProps) {
     const [reportIDsList = getEmptyArray<string>()] = useOnyx(ONYXKEYS.TRANSACTION_THREAD_NAVIGATION_REPORT_IDS, {
         canBeMissing: true,
     });
 
-    const {prevReportID, nextReportID} = useMemo(() => {
+    const [parentReportActions = new Map<string, string>()] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${parentReportID}`, {
+        canBeMissing: true,
+        selector: (reportActions) => {
+            // Map of childReportID => parentReportActionID
+            const parentActions = new Map<string, string>();
+            for (const action of Object.values(reportActions ?? {})) {
+                if (!action.childReportID) {
+                    continue;
+                }
+                parentActions.set(action.childReportID, action.reportActionID);
+            }
+            return parentActions;
+        },
```

Comment:
We should avoid inline selectors. Can you please put it out of the component to create a stable reference? cc @TMisiukiewicz here is some more context https://github.com/Expensify/App/issues/70272

Thanks!

Reply:
Done